script.module.pyhdhr/lib/pyhdhr/PyHDHR.py

```diff
@@ -29,22 +29,28 @@ def __init__(self):
         return
```

Comment:
probably you might consider to use a custom logger that extends `logging.StreamHandler`. See for instance: https://github.com/xbmc/generator-kodi-addon/blob/master/generators/app/templates/resources/lib/kodilogging.py

In fact this could even be turned into a script.module since I am also using it in my addons

Reply:
Thanks I'll take a look.